web/src/beta/features/Visualizer/Crust/Infobox/Block/builtin/Markdown/index.tsx

```diff
@@ -26,8 +26,12 @@ const MarkdownBlock: FC<CommonBlockProps<InfoboxBlock>> = ({
   );
 
   const evaluatedSrc = useExpressionEval(src);
-  const sketchCustomPropertyNames: string[] | undefined =
-    useSketchCustomPropertyNames(layer);
+  const propertyNames = Object.keys(selectedFuture?.properties).filter(
+    (key) => {
+      const defaultProperty = ["extrudedHeight", "id", "positions", "type"];
+      return !defaultProperty.includes(key);
+    }
+  );
```

Comment:
Should we check `selectedFeature?.properties`? Since `Object.keys(undefined)` could throw out error i think.